tests/integration/project-components/project-list-entry-skeleton.test.ts

```diff
@@ -0,0 +1,24 @@
+import { expect, test, describe } from "vitest";
+import ProjectListEntrySkeleton from "$lib/components/composites/project-components/ProjectListEntrySkeleton.svelte";
+import { render, screen } from "@testing-library/svelte";
+
+describe("ProjectListEntrySkeletonComponent", () => {
+    test("When project list entry skeleton is rendered, then no data are displayed but only skeletons.", () => {
+        render(ProjectListEntrySkeleton);
+
+        // expect skeleton for project name, members and stage, but not progress
+        screen
+            .getAllByTestId("skeleton", { exact: false })
+            .forEach((skeleton) => expect(skeleton).toBeInTheDocument());
```

Comment:
Isn't this redundant? If you can fetch them with `getAllByTestId`, they have to be in the document.